.github/bin/modified-workspaces/exec.mjs

```diff
@@ -0,0 +1,40 @@
+import { spawn } from 'child_process';
+
+// Run a child proc
+// - throws on non-zero exit codes
+// - returns { stderr: ..., stdout: ... } when successful
+// Is mainly a more ergonomic API for `spawn`.
+export async function exec(cmd, args) {
+	const result = await new Promise( ( resolve, reject ) => {
+		const childProc = spawn(
+			cmd,
+			[...args],
+		);
+
+		let stdout = '';
+		let stderr = '';
+
+		childProc.stdout.on( 'data', ( data ) => {
```

Comment:
I'm not against and probably nitpicky but this code style from WordPress doesn't feel consistent. Is this something you want to enforce?

```suggestion
export async function exec(cmd, args) {
	const result = await new Promise((resolve, reject) => {
		const childProc = spawn(
			cmd,
			[...args],
		);

		let stdout = '';
		let stderr = '';

		childProc.stdout.on('data', (data) => {
```

Note that this suggestion is not complete and the whole file seems to have mixed styles. Just curious about this.